parlai/core/params.py

```diff
@@ -1000,6 +1000,83 @@ def parse_args(self, args=None, namespace=None, print_args=True):
 
         return self.opt
 
+    def _kwargs_to_strargs(self, **kwargs):
```

Comment:
just for my understanding, this function takes a dict of args and converts to what you'd write on the command line?

Reply:
yup